src/js/view/controls/controlbar.js

```diff
@@ -334,10 +334,9 @@ export default class Controlbar {
         }
 
         if (elements.cast && elements.cast.button) {
-            const castUi = elements.cast.ui.on('click', function(evt) {
-                // controlbar cast button needs to manually trigger a click
-                // on the native cast button for taps and enter key
-                if (evt.sourceEvent.detail !== 1) {
+            const castUi = elements.cast.ui.on('click enter', function(evt) {
```

Comment:
Can we use `addClickAction` here?

Reply:
The UI gets created by the `createCastButton` function, which is called in a scope that doesn't have access to the values in this closure.